sdk/python/tests/test_repo_operations.py

```diff
@@ -5,8 +5,19 @@
 from typing import Optional
 
 import assertpy
+import pytest
+from fixtures.data_source_fixtures import simple_dataset_1  # noqa: F401
```

Comment:
is there a reason for the `noqa`?

Reply:
the linter wants to group the imports from the same module but when i use parametrize, it wants the import to be a separate line. Since i no longer am going to use parametrize as per ur other comment, we won't have a noqa here, but it may be need again when importing fixtures in future with use with parametrize

Reply:
perhaps try https://stackoverflow.com/questions/47196947/py-test-deal-with-both-pylint-and-flake8-when-importing-features-from-a-module

Reply:
ah got it